libraries/abstractions/mqtt_agent/freertos_agent_message.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * FreeRTOS V202011.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Are these abstraction files under `mqtt_agent` folder exactly the same as those in `FreeRTOS/FreeRTOS` [here](https://github.com/FreeRTOS/FreeRTOS/tree/202104.00/FreeRTOS-Plus/Demo/Common/coreMQTT_Agent_Interface)?

Reply:
Not exactly. They were taken from the demos repo https://github.com/FreeRTOS/coreMQTT-Agent-Demos/tree/main/lib/FreeRTOS/mqtt-agent-interface. Besides the headers the code is the same